chimba/chain.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "chimba",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://chimba.ooo/",
+  "pretty_name": "Chimba",
+  "chain_id": "chimba",
+  "bech32_prefix": "chimba",
+  "daemon_name": "chimbad",
+  "node_home": "$HOME/.chimbad",
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "chimba",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 1.0,
+        "average_gas_price": 5.0,
+        "high_gas_price": 10.0
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "chimba"
+      }
+    ]
+  },
+  "codebase": {
```

Comment:
this codebase information is for chihuahua chain. Please correct this to reflect chimba chain